incubator/binding-openapi/pom.xml

```
@@ -173,7 +173,7 @@
                   </fileMappers>
                 </artifactItem>
               </artifactItems>
-              <includes>io/aklivity/zilla/specs/binding/openapi/schema/openapi.schema.patch.json</includes>
+              <includes>io/aklivity/zilla/specs/binding/openapi/schema/*.json</includes>
```

Let's make this explicit, as we might decide to move the supporting openapi version-specific schemas to a different directory.

```suggestion
              <includes>io/aklivity/zilla/specs/binding/openapi/schema/openapi.schema.patch.json</includes>
              <includes>io/aklivity/zilla/specs/binding/openapi/schema/openapi.*.schema.json</includes>
```